client/state/index.js

```diff
@@ -136,6 +136,7 @@ export function createReduxStore( initialState = {} ) {
 		isBrowser && require( './happychat/middleware.js' ).default(),
 		isBrowser && require( './analytics/middleware.js' ).analyticsMiddleware,
 		isBrowser && require( './data-layer/wpcom-api-middleware.js' ).default,
+		isBrowser && require( './data-layer/extensions-middleware.js' ).default,
```

Comment:
The only other context really is SSR, right? Given that most of the data `extensions-middleware` will deal with is API data from other places, do we want to do that server-side before we serve up the page? It sounds like more complication that maybe we don't need.